testnet/stacks-node/src/tests/signer.rs

```diff
@@ -1266,3 +1322,72 @@ fn stackerdb_filter_bad_transactions() {
     }
     signer_test.shutdown();
 }
+
+#[test]
+#[ignore]
+/// Test that signers will be able to continue their operations even if one signer is restarted.
+///
+/// Test Setup:
+/// The test spins up three stacks signers, one miner Nakamoto node, and a corresponding bitcoind.
+/// The stacks node is advanced to epoch 2.5, triggering a DKG round. The stacks node is then advanced
+/// to Epoch 3.0 boundary to allow block signing.
+///
+/// Test Execution:
+/// The signers sign one block as usual.
+/// Then, one of the signers is restarted.
+/// Finally, the signers sign another block with the restarted signer.
+///
+/// Test Assertion:
+/// The signers are able to produce a valid signature after one of them is restarted.
+fn stackerdb_sign_after_signer_reboot() {
```

Comment:
Please add this to the `bitcoin-tests.yaml` file so it runs in CI

Reply:
Thanks for noting. Will do.

Reply:
Done :heavy_check_mark: